lib/projects/localDev/LocalDevWatcher.ts

```diff
@@ -0,0 +1,70 @@
+import path from 'path';
+import chokidar, { FSWatcher } from 'chokidar';
+
+import { PROJECT_CONFIG_FILE } from '../../constants';
+import LocalDevProcess from './LocalDevProcess';
+
+const WATCH_EVENTS = {
+  add: 'add',
+  change: 'change',
+  unlink: 'unlink',
+  unlinkDir: 'unlinkDir',
+};
+
+class LocalDevWatcher {
```

Comment:
This is a good example of how external sources can interact with the `LocalDevProcess`